hw/ip/aes/data/aes.hjson

```diff
@@ -6,8 +6,8 @@
 {
   name: "aes",
   clocking: [
-    {clock: "clk_i", reset: "rst_ni", primary: true},
-    {clock: "clk_edn_i", reset: "rst_edn_ni"}
+    {clock: "clk_i", reset: "rst_ni", idle: "idle_o", primary: true},
+    {clock: "clk_edn_i", reset: "rst_edn_ni", idle: "idle_o"}
```

Comment:
would we be able to mark only 1 of the clocks with an idle but leave the other vanilla? 
Just in case in the future we need to push escalate / alert net work clocks into modules, the latter group would be better if it's "untouched".

Reply:
The way I've written the topgen code in this PR, it expects that if you connect a hint clock to multiple clock ports then both of those clock ports have to point at the same idle signal. At the moment, AES has `clk_i` and `clk_edn_i` both connected to the same hint clock ("main").

I feel like the hjson above is nice and clear, saying: "This block expects `clk_i` and `clk_edn_i` to keep running unless `idle_o` is asserted" (which is a statement about the internals of the AES block; when connecting things up, topgen then checks that the statement will hold).

Can you think of another way to do this clearly?